src/main/java/de/progen_bot/listeners/StarBoardListener.java

```diff
@@ -0,0 +1,75 @@
+package de.progen_bot.listeners;
+
+import java.awt.Color;
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+import de.progen_bot.core.Main;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.TextChannel;
+import net.dv8tion.jda.api.events.message.guild.react.GuildMessageReactionAddEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+
+public class StarBoardListener extends ListenerAdapter {
+    private final Map<String, Message> starMessage = new HashMap<>();
+    private final Map<String, Integer> starCount = new HashMap<>();
```

Comment:
Storing JDA entities is not recommended due to random cache invalidation. Try retrieving message by stored IDs

Reply:
Yes good point!